src/blog/2023/03/3-quick-node-red-tips-4.md

```diff
@@ -0,0 +1,52 @@
+```
+
+### 2. Perform simple maths functions using JSONata in Change nodes
+
+You can perform basic maths functions using the Change node and JSONata. Let's say you wanted to take a payload and multiply it by a value. You could use a custom node such as [node-red-contrib-calc](https://flows.nodered.org/node/node-red-contrib-calc) but you can also easily complete the same task within a change node. 
```

Comment:
```suggestion
You can perform basic maths functions using the Change node and JSON data. Let's say you wanted to take a payload and multiply it by a value. You could use a custom node such as [node-red-contrib-calc](https://flows.nodered.org/node/node-red-contrib-calc) but you can also easily complete the same task within a change node. 
```

Reply:
As above, it's called JSONata, it's a play on the musical term Sonata.

Reply:
OK, I should have figure it out.